app/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,7 @@
         -->
         <meta-data
             android:name="com.google.android.geo.API_KEY"
-            android:value="${YOUR_API_KEY}" />
+            android:value="" />
```

Comment:
Made this value blank so program can compile for people without a GoogleMaps API key.  Setup for this key isn't possible without enabling billing to your google account (entering your credit card info).

Reply:
I kinda think it's good to leave it there, so you remember what it's for.  But... there's a bigger concept here.  API Keys should not be pushed to GitHub, as they are publicly viewable.  Consider a way to keep this API key hidden.  There are some tricks to doing that, so that the API key is stored in a variable, and the actual value is never submitted to GitHub.